### PR TITLE
docs: add youtube to social icons

### PR DESCRIPTION
**Pull request by twerske**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
YouTube channel is linked in the socials

Screen Shot 2021-02-24 at 12 49 48 PM

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by mgechev — APPROVED** [review]

**Review by gkalpak — APPROVED** [review]

File: aio/src/app/app.component.html [hunk 1]
```diff
@@ -31,6 +31,9 @@
       <a href="https://github.com/angular/angular" title="GitHub" aria-label="Angular on github">
         <mat-icon svgIcon="logos:github"></mat-icon>
       </a>
+      <a href="https://youtube.com/angular" title="YouTube" aria-label="Angular on YouTube">
+        <mat-icon svgIcon="logos:youtube"></mat-icon>
+      </a>
```
Comment (gkalpak) [inline aio/src/app/app.component.html:36]: OOC, is the order of the icons deliberate (or did you just add the new icon at the end)?

**Comment by gkalpak** [comment]
@googlebot I consent.

**Comment by gkalpak** [comment]
I rebased on master (to resolve a CI issue) and added fixup commit with a minor suggestion.
Leaving it to you to add the `action: merge` label to give you a chance to look at my comments.

File: goldens/size-tracking/aio-payloads.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "master": {
       "uncompressed": {
         "runtime-es2015": 3153,
-        "main-es2015": 433285,
+        "main-es2015": 434502,
```
Comment (gkalpak) [inline goldens/size-tracking/aio-payloads.json:24]: Not sure why, but it seems that the size increase in ViewEngine is twice as larger as the one in Ivy 🤷‍♂️

**Comment by gkalpak** [comment]
I pushed another commit to update the payload sizes (since [CI was failing][1]).

[1]: https://app.circleci.com/pipelines/github/angular/angular/28914/workflows/b1aa20c1-818d-498e-bc8d-406ae37e1e17

**Review by jessicajaniuk — APPROVED** [review]
LGTM 🍪 

Reviewed-for: size-tracking

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: size-tracking

File: aio/src/app/app.component.html (hunk 1 above)
Comment (twerske) [reply aio/src/app/app.component.html:36]: added at the end as least prioritized

**Comment by zarend** [comment]
hi @twerske 👋 . I'm unable to merge this because it has merge conflicts with master.

Could you please rebase this branch is `upstream/master` to unblock merging it?

**Comment by zarend** [comment]
[presubmit](http://test/OCL:359808070:BASE:359808102:1614369595662:fd863b2e)

**Pull request closed without merging**